gammapy/modeling/models/spatial.py

```diff
@@ -439,22 +450,34 @@ def evaluate(lon, lat, lon_0, lat_0, sigma, e, phi):
         exponent = -0.5 * ((1 - np.cos(sep)) / a)
         return u.Quantity(norm * np.exp(exponent).value, "sr-1", copy=False)
 
-    def to_region(self, **kwargs):
-        """Model outline (`~regions.EllipseSkyRegion`)."""
+    def to_region(self, n_sigma=1, **kwargs):
+        """Model outline at a given number of :math:`\sigma`.
+        
+        Parameters
+        ----------
+        n_sigma : float
```

Comment:
This is nitpicky: `n_sigma` suggests it would be an integer and not a float, maybe there is a different name...